UnitTests/Tests/Formatters/TRTemperatureComparisonFormatterSpec.m

```diff
@@ -0,0 +1,34 @@
+#import "UnitTests-Swift.h"
+#import "TRTemperatureComparisonFormatter.h"
+
+@interface TRTemperatureComparisonFormatter (Tests)
+
++ (NSCalendar *)calendar;
+
+@end
+
+SpecBegin(TRTemperatureComparisonFormatter)
+
+NSDate* (^dateFromString) (NSString*) = ^NSDate* (NSString *dateString) {
```

Comment:
Actually, we should have Quick/Nimble/Swift in master at this point, right? Maybe we can test this whole thing in Swift and not have to deal with this at all?